modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/CreateAnAPIApplicationEndpoint.testcase

```diff
@@ -93,6 +115,274 @@ definition {
+
+			AssertElementPresent(
+				fieldLabel = "Scope",
+				locator1 = "FormViewBuilder#FIELD_LABEL");
+
+			AssertElementPresent(
+				fieldLabel = "Path",
+				locator1 = "FormViewBuilder#FIELD_LABEL");
+		}
+	}
+
+	@priority = 4
+	test CanSeeAllFieldsInCreationModal {
+		property portal.acceptance = "true";
+
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
+		}
+
+		task ("When I click 'Add API Endpoint' button") {
+			Click(locator1 = "APIBuilder#ADD_API_ENDPOINT");
+		}
+
+		task ("Then fields Scope *, Path * and Description are present") {
+			AssertElementPresent(
+				fieldLabel = "Scope",
+				locator1 = "FormViewBuilder#FIELD_FROM_FIELDSET");
+
+			AssertElementPresent(
+				fieldLabel = "Path",
+				locator1 = "FormViewBuilder#FIELD_FROM_FIELDSET");
+
+			AssertElementPresent(
+				fieldLabel = "Description",
+				locator1 = "FormViewBuilder#FIELD_FROM_FIELDSET");
+		}
+	}
+
+	@priority = 3
+	test CanSeeInfoOfCreatedEndpointWithDescription {
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
+		}
+
+		task ("When I enter a path and Scope to create the API endpoint") {
+			APIBuilderUI.createAPIEndpoint(
+				description = "Description default",
+				path = "testendpoint2",
+				retrieveType = "Collection",
+				scope = "Company");
+		}
+
+		task ("Then I can see success alert") {
+			Alert.viewSuccessMessage();
+		}
+
+		task ("And Then I can see info of created endpoint with 'Get' method, scope, path") {
+			AssertElementPresent(
+				key_value = "GET",
+				locator1 = "APIBuilder#DROPDOWN_TOGGLE");
+
+			AssertElementPresent(
+				key_value = "Company",
+				locator1 = "APIBuilder#DROPDOWN_TOGGLE");
+
+			AssertElementPresent(
+				key_fieldLabel = "Path",
+				locator1 = "TextInput#GENERIC_TEXT_INPUT",
+				value1 = "testendpoint2");
+
+			AssertElementPresent(
+				key_value = "endpoint",
+				locator1 = "APIBuilder#DESCRIPTION",
+				value1 = "Description default");
+		}
+	}
+
+	// @description = "Case by LPS-197668 | Every time the test runs, it creates a new instance and therefore, it is not possible to assert in 'localhost-f0g7f0h7:8080/o/c/test2' you need to create a macro for this"
+
+	@ignore = "true"
+	@priority = 4
+	test CanSeePathOfPathFields {
+		property portal.acceptance = "true";
+
+		task ("And Given I go to 'Endpoints' tab in 'Edit API Application'") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Endpoints",
+				title = "test2");
+		}
+
+		task ("When I click 'Add API Endpoint' button") {
+			Click(locator1 = "APIBuilder#ADD_API_ENDPOINT");
+		}
+
+		task ("Then I can see path under {portalUrl}/o/c/{applicationName}") {
+			AssertTextEquals(
+				locator1 = "APIBuilder#URL_LABEL",
+				value1 = "${portalURL}/o/c/test2");
```

Comment:
Hi @pedroamarall, for ${portalURL}, I think you can try to use '`JSONCompany.getPortalURL();`' to get current baseURL